jdbc-driver/src/test/java/com/google/cloud/dataproc/jdbc/DataprocInfoTest.java

```diff
@@ -15,26 +15,22 @@
  */
 package com.google.cloud.dataproc.jdbc;
 
+import static com.google.cloud.dataproc.jdbc.DataprocInfo.DATAPROC_INTERCEPTOR;
+import static com.google.cloud.dataproc.jdbc.DataprocInfo.HTTP_INTERCEPTOR;
```

Comment:
Similarly, you could just inline these constants in the test's jdbc url: `;http.interceptor=...`